portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsFilterBreakdown.testcase

```diff
@@ -133,10 +133,13 @@ definition {
 	}
 
 	@description = "Feature ID: LRAC-7868 | Automation ID: LRAC-10265 | Test Summary: Max of 3 breakdowns options are allowed"
+	@ignore = "true"
```

Comment:
Hey Vini, I don't understand a few things:
- why are you adding the ignore property in some cases (3 cases)?
- since you are adding the ignore, why are you changing the number of attributes since the test won't run anyway?

Reply:
Hey @manuferreira, I ignored these 3 tests because their objective is to use more than one breakdown to do the validation, so the test needs to add more than one breakdown for that, if I change the test I will modify the objective, so I decide to disable the test. These 3 tests I just deactivated and put a comment, I didn't change the breakdown number in these cases.

Reply:
ah you're right, I was reading the changes and I thought it was associated with those 3 tests you turned off :grin: anyway, thank you